tests/acceptance/expectUserAndAdminLoginCest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/*
+ Cest is a class based format. Cest format is able to support multiple 
+ tests per file and easy reuse of code by adding additional private functions.
+ Cest combines scenario-driven test approach with OOP design. 
+ */
+class expectUserAndAdminLoginCest
+{
+    public function _before(AcceptanceTester $I)
+    {
+        //runs before each test
+        $I->amOnPage('/');
+    }
+    public function _after(AcceptanceTester $I)
+    {
+        //runs after each test
+    }
+    // tests
+    
+    public function expectsUserToLogin(AcceptanceTester $I) {
+        $I->wantTo('log in as a user');
+        $I->loginAs('chappress', 'password');
+    }
+    public function expectsAdminToLogin(AcceptanceTester $I) {
+        $I->wantTo('log in as an Admin');
+        $I->loginAsAdmin();
+        $I->amOnPage('/wp-admin/index.php');
+    }
+    function expectsAddNewUser(AcceptanceTester $I)
```

Comment:
Yes, php defaults to public for methods. I don't really care which way we do it so much as we're consistent about it.

But for questions like this, let's refer to this (until we get our own style guide page):

- http://www.phptherightway.com/